### ISSUE TITLE
Please add support for iOS 15.7

### ISSUE THREAD

**Issue by hardik99** [jijiucheng/JJC_iOS_DeviceSupport#1]

**Comment by jijiucheng** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by jijiucheng** [comment]
The device support file for iOS 15.7 have been added and are available in two versions, extracted from Xcode 13.4.1 and Xcode 14, it is recommended that you use the device support file extracted from Xcode 14.

Note:
According to the Apple - Xcode Release Notes released for Xcode 14, it is mentioned that:
[Xcode 14 release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes)

```
Xcode 14 cannot be used with iOS 15.7 for development. (99847608)
Workaround: Use Xcode 13.4.1 with iOS 15.7.
```
